### PR TITLE
Automation of tool evaluation

### PR DESCRIPTION
**Pull request by tink-expo**
- Currently just use console stdout to log test result. To use log file for saving test results, please use
```python tool_eval.py > log.txt```.
- Default behavior is running tool evaluation with all ```.py``` files in ```manual_src``` dir (manually created sources by @SeonghaEom), once per single file. 
For more argument options, please refer to ```python tool_eval.py -h```.
- ```-g $COUNT``` option supports testing with python file automatically generated by ```gen_src```. 
However, the range of syntaxes ```gen_src``` generates doesn't seem to sync completely with range of syntax supportable by current AST parser. This issue should be revisited.

### CODE REVIEW

**Comment by tink-expo** [comment]
This PR is related to slack discussion [here](https://cs453-2020.slack.com/archives/G0101830UH2/p1591702124064000)

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import setuptools
+
+# install package with 'python3 setup.py install'
+
+setuptools.setup(
+    name="gen_src",
+    version="2",
+    author="Woohyun Han",
```
Comment (tink-expo) [inline setup.py]: @SeonghaEom Please feel free to add you here XD

Comment (SeonghaEom) [reply setup.py]: You are angel + genius.. 😊

**Review by ehdkacjswo — APPROVED** [review]
👍👍👍

**Pull request merged**